sdk/synapse/azure-analytics-synapse-artifacts/src/main/java/com/azure/analytics/synapse/artifacts/KqlScriptsClient.java

```diff
@@ -29,6 +30,34 @@ public final class KqlScriptsClient {
         this.serviceClient = serviceClient;
     }
 
+    /**
+     * Get all KQL scripts.
+     *
+     * @throws ErrorContractException thrown if the request is rejected by server.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     * @return all KQL scripts along with {@link PagedResponse}.
+     */
+    @Generated
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public PagedResponse<KqlScriptResource> getAllSinglePage() {
+        return this.serviceClient.getAllSinglePage();
+    }
+
+    /**
+     * Get all KQL scripts.
+     *
+     * @param context The context to associate with this operation.
+     * @throws IllegalArgumentException thrown if parameters fail the validation.
+     * @throws ErrorContractException thrown if the request is rejected by server.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     * @return all KQL scripts along with {@link PagedResponse}.
+     */
+    @Generated
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public PagedResponse<KqlScriptResource> getAllSinglePage(Context context) {
+        return this.serviceClient.getAllSinglePage(context);
+    }
```

Comment:
these should not be exposed.

Reply:
solved via https://github.com/Azure/autorest.java/pull/1723/commits/2b1c3fa1dc42da3797a8b637c3fc4cd483d13e25